vendor/priors/gwt.js

```diff
@@ -74,7 +74,12 @@ function postProm(url, data) {
             }
             else {
 		try {
-		    reject(req.getResponseHeader("Trailer"));
+            let trailer = req.getResponseHeader("Trailer");
+            if (trailer.startsWith('Storage+quota+reached')) {
+                future.completeExceptionally(new peergos.shared.storage.StorageQuotaExceededException(trailer));
```

Comment:
will this show it url decoded eventually to the user?

Reply:
yes